File: corpus/typescript/fixture_01.ts
type CartItem1 = { price: number; qty?: number };
export function calculateDiscount1(cartItems: CartItem1[], taxRate: number): number {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 10 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook1 {
  lookupPrice(sku: string): number { return sku.length + 1; }
}

File: corpus/python/fixture_01.py
def calculate_discount_1(cart_items, tax_rate):
    total_amount = sum(item["price"] * item.get("qty", 1) for item in cart_items)
    if total_amount > 10:
        return total_amount * (1 + tax_rate)
    return total_amount

class PriceBook1:
    def lookup_price(self, sku):
        return len(sku) + 1
